benzina/src/typed_uuid.rs

```diff
@@ -88,6 +92,13 @@ macro_rules! typed_uuid {
             $vis struct $name(::uuid::Uuid);
 
             impl $name {
+                /// Creates a new, unchecked typed Uuid
+                #[must_use]
+                #[cfg(feature = "test")]
```

Comment:
I'd name this feature something like `dangerous-construction`

Reply:
I like it!